apps/ui/src/components/ui/carousel.tsx

```diff
@@ -122,7 +122,7 @@ export function Carousel({
                 alt={image.alt}
                 //fill
                 sizes="100vw"
-                className={`${cover ? "object-cover" : "object-contain"}`}
+                className={`${cover ? "object-cover" : "object-contain"} h-full w-full`}
```

Comment:
I added h-full and w-full classes here so that the images would fit the carousel's container. I noticed that the Image component from next is not being imported anymore and a simple <img is being used instead. Next's image has a `full` property that I believe can be used in place of h-full w-full if you plan on using next's component instead.